src/Air.zig

```diff
@@ -1521,40 +1516,56 @@ pub fn deinit(air: *Air, gpa: std.mem.Allocator) void {
     air.* = undefined;
 }
 
-pub const ref_start_index: u32 = InternPool.static_len;
+pub inline fn refToInternedAllowNone(ref: Inst.Ref) ?InternPool.Index {
```

Comment:
Can you justify the use of `inline` in these newly added functions?

Reply:
I don't have a strong justification - I like marking trivial functions like this as `inline` just since it takes that work off of the optimizer (particularly relevant once we start doing more optimizations in self-hosted backends; something like `refToInterned(ref).?` is easily optimizable if the call is inlined) and I highly doubt this impacts analysis speed in any measurable way (these functions aren't called *that* much after all). But I can remove it and rely on the optimizer if you'd prefer

Reply:
Yes please. In general, let's avoid limiting the options the compiler has available to it unless there is a reason for it - which in this case I don't think clearly exists.